install_sct

```diff
@@ -599,13 +599,11 @@ python/bin/conda create -y -n venv_sct python=3.7
 find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +
 
 # make sure that there is no conflict with local python install by making venv_sct an isolated environment.
-# workaround for https://github.com/conda/conda/issues/7173
+# The fix comes from https://docs.python.org/3/library/site.html#module-site
```

Comment:
Those docs aren't very clear at all. I didn't actually figure it out by reading them, but my looking at what `python -m venv` did and working backwards to find the matching part in site.py. Maybe this is a more helpful explanation then?

```suggestion
# The fix is based on the way virtualenv works as explained
# in https://docs.python.org/3/library/site.html#module-site
```

Reply:
It bugs me that the docs say

> If “pyvenv.cfg” (a bootstrap configuration file) contains the key “include-system-site-packages” set to anything other than “true” (case-insensitive), the system-level prefixes will not be searched for site-packages; otherwise they will.

but it really means "system-site" (`/usr/lib/python*/site-packages`) AND user site (`~/.local/lib/python*/site-packages`), and also that it uses that double-negative.

Reply:
Oh, absolutely! That's also why I linked to your summary from the `conda` issue:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/b42e007b0993266c585c0aee64ae5fd7300e5e63/install_sct#L605

But perhaps that could be made clearer, e.g.:

```python
# see also
# * Issue details: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3067
# * Fix details: https://github.com/conda/conda/issues/7173#issuecomment-980496682
```

Reply:
I ended up just deleting the link to the Python docs, since your conda comment links it too. Probably better to just avoid the misleading page in the first place and show your explanation right from the start.